### ISSUE TITLE
Remove quotation marks?

### ISSUE THREAD

**Issue by JensRantil** [adamgordonbell/csvquote#3]
Hello! I love this tool! I thought of a small potential improvement:

If the intention is that I am supposed to be able to work with CSV fields in `awk`, `sed` etc. one problem I'm hitting is that I need to deal with the actual quotation marks wrapping the columns. Would it be possible to skip the quotation marks around the fields and then readd them with `csvquote -u` (if needed?)?

**A problem/usecase:** I have a CSV file containing a file of financial transactions. It looks likes this:
```csv
date,description,amount
"2022-06-02","McDonalds","22"
"2022-06-02","Burger King","42"
```
Unfortunately I'm not in control of the format of this file - that is I can't control the quotation. I would like to negate the amounts. My initial take was to do this with `cat file.csv | csvquote | awk '{$3=-$3;print}' | csvquote -u`. However, this doesn't work as `$3` in awk is `"22"`, not `22`.

**Workaround:** I can of course do `gsub(/"/, "", $3);$3=-$3;$3="\"" $3 "\"";` or something but it's pretty hacky. I could also of pipe my file through [`csvformat`](https://csvkit.readthedocs.io/en/latest/scripts/csvformat.html) which I _think_ should use minimal quoting, but it would be really nice to not have to do that.

**Proposal:** Remove the output quotations because I _think_ they aren't needed. Readd them when doing `csvquote -u` again (if needed?).

Does this make sense?

**Comment by adamgordonbell** [comment]
Hey @JensRantil  I see what you are saying.  Apologies on the delay.

Couldn't there be cases where removing quotes would be unexpected?, even if added back in the end?

I think i would try, in your example, to have awk split on the quotes.
```
 awk -F'"' '{$3=-$3;print}' 
```

**Comment by JensRantil** [comment]
> Couldn't there be cases where removing quotes would be unexpected?, even if added back in the end?

I guess one unexpected thing would be that

```
$ echo '"hello","there"' | csvquote | csvquote -u
```
would yield `hello,there` instead of `"hello","there"`. That's the only corner case I can up with right now. 🤔

**Comment by adamgordonbell** [comment]
I'd like to keep csvquote so it it always gives you the same output as input. It makes it easy to reason about. 

So PRs welcome, but it should maintain that invariant.  

If csvformat does the minimal quoting, then I think that might be the way to go.

**Comment by JensRantil** [comment]
Yeah, it makes sense the inverse should give the initial output. 👍 A slightly different approach would be to replace quotation marks with 0x1D and then have them be replaced back on `csvquote -u`... 🤔
